include/odp/api/spec/packet_io.h

```diff
@@ -412,6 +412,9 @@ typedef union odp_pktout_config_opt_t {
 		/** Insert SCTP checksum on packet by default */
 		uint64_t sctp_chksum     : 1;
 
+		/** Packet references disabled */
+		uint64_t no_packet_refs  : 1;
```

Comment:
This part needs more description here in the API spec (user guide is not spec but just additional information how to operate the API). Something like this would be needed:

"Packet references not used on packet output

When set, application indicates that it will not transmit packet references on this packet IO interface.

0: Packet references may be transmitted on the interface (the default value).
1: Packet references will not be transmitted on the interface "